__tests__/app.test.js

```diff
@@ -33,4 +34,62 @@ describe("API Testing", () => {
       });
     });
   });
+  describe("/api/reviews", () => {
+    describe.only("GET requests", () => {
+      test("Should return a status code of 200", () => {
+        return request(app).get("/api/reviews").expect(200);
+      });
+      test("Should return an object containing an array of reviews", () => {
+        return request(app)
+          .get("/api/reviews")
+          .expect(200)
+          .then((res) => {
+            res.body.reviews.forEach((obj) => {
```

Comment:
watch out here, as your not testing the length of the returned array in any way, which means we could have passed this test with an empty array :)